index.html

```diff
@@ -1545,6 +1545,46 @@
 </p>
 </div>
 
+### Guardians {#guardians}
+
+Some classes of vulnerable people tend to be unable to make good decisions about their own web use,
+and need a <dfn>guardian</dfn> to help them. Children are a widely recognized example of this class,
+with their parents commonly acting as their [=guardians=].  A person with a [=guardian=] is known as
+a <dfn>ward</dfn>. In some limited ways, employees with valuable access to systems can be seen as
+[=wards=] of their IT departments.
+
+Many legal systems treat these guardianship relationships as a set of rights that the [=guardian=]
+possesses. We prefer to instead think of the [=ward=] having a right to make informed decisions and
+exercise their autonomy. Their [=guardian=] then has an _obligation_ to help their [=ward=] do so
+when the [=ward=]'s abilities aren't sufficient, even if that conflicts with the [=guardian=]'s
+desires. In practice, many [=wards=] discover that their [=guardian=] is not making decisions in the
+[=ward=]'s best interest, and it's critical that such [=wards=] have a way to escape their
+misbehaving guardian.
+
+Historically, the Web has provided exactly this escape route, and [=user agents=] should preserve
+that feature by correctly balancing a benevolent [=guardian=]'s need to protect their [=ward=] from
+dangers against other [=wards=]' need to protect themselves from their dangerous [=guardians=].
+
+<aside class="example" id="example-protective-parent" title="Protective parents">
+
+A parent might configure a small child's [=user agent=] to block access to violent content until the
+child is old enough to make their own decisions about it.
```

Comment:
@torgo, I feel like you had a good example of this sort of protection. What was it?